submodules/initial_validation/README.md

```diff
@@ -0,0 +1,45 @@
+# Submodule initial-validation
```

Comment:
I think this logic would fit better within the root module - see example of the pattern at https://github.com/terraform-ibm-modules/terraform-ibm-resource-group/blob/a48ba6581c714d31aca070d330bd9310141ad977/main.tf#L8-L15

Reply:
main.tf is not the right place. The workflow should fail in terraform plan, if input variables do not fit to each other. Decided to make a separate module and include all the logic to verify input + environments BEFORE something will be created.

Reply:
The logic above would cause to fail at terraform plan

Reply:
Right, But it is good to keep it in a separate module which tracks all variable validation right ?

Reply:
@surajsbharadwaj - It's the same. If you want to stick with a different module for validation - could you just remove the empty tf files (main, output)